src/js/locks.js

```diff
@@ -52,7 +52,8 @@ define(function (require, exports, module) {
         JS_STYLE: "jsStyle",
         JS_LIBRARIES: "jsLibraries",
         CC_LIBRARIES: "ccLibraries",
-        OS_CLIPBOARD: "osClipboard"
+        OS_CLIPBOARD: "osClipboard",
+        PS_GENERATOR: "psGenerator"
```

Comment:
I actually am not at all sure it is necessary even to have a generator lock since.  I'd welcome your thoughts on this matter.  In the immediate term, I will apply your suggested changes.